CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Release Notes
 
-## Platform Version 0.10.3 - Unreleased
+## Platform Version 0.10.3 - 2022-12-16
+* Add `fluvio cluster status` ([#2824](https://github.com/infinyon/fluvio/issues/2824))
+* Display fetch status ([#2872](https://github.com/infinyon/fluvio/issues/2872))
+* Emphasize flush in docs ([#2850](https://github.com/infinyon/fluvio/issues/2850))
```

Comment:
bunch of other changes are not reflected here

Reply:
Added entry for smdk fix and for smartengine metrics fix.